sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOSiteVersionPolicyJob.md

```diff
@@ -0,0 +1,68 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/remove-spositeversionpolicyjob
+applicable: SharePoint Online
+title: Remove-SPOSiteVersionPolicyJob
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Remove-SPOSiteVersionPolicyJob
+
+## SYNOPSIS
+
+Cancels further processing of setting version policy job for existing document libraries on the site collection.
+
+## SYNTAX
+```powershell
+Remove-SPOSiteVersionPolicyJob [-Identity] <SpoSitePipeBind>
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Set-SPOSite using ApplyToExistingDocumentLibraries will create a job to set version policy for existing document libraries on the site collection. Remove-SPOSiteVersionPolicyJob is to remove the job. This will stop updating the new version policy on existing libraries. The version policy already applied on existing libraries will not be reverted.
```

Comment:
>Remove-SPOSiteVersionPolicyJob is to remove the job. This will stop updating the new version policy on existing libraries [](http://example.com/codeflow?start=147&length=121)

"This cmdlet removes the job, which stops updating the new version policy on existing libraries."